src/Microsoft.DotNet.Interactive.Tests/LanguageKernelPackageTests.cs

```diff
@@ -116,41 +116,94 @@ public async Task it_can_load_assembly_references_using_r_directive_separate_sub
+        [InlineData(Language.FSharp, false)]
+        [InlineData(Language.CSharp, true)]
+        [InlineData(Language.FSharp, true)]
+        public async Task it_can_load_script_files_using_load_directive_with_relative_path(Language language, bool changeCurrentDirectoryInUserCode)
+        {            
+            var currentDirectory = Directory.GetCurrentDirectory();
+            DisposeAfterTest(() => Directory.SetCurrentDirectory(currentDirectory));
+
+            var kernel = CreateKernel(language);
+
+            if (changeCurrentDirectoryInUserCode)
             {
-                Language.CSharp => $"#r \"{relativePath}\"",
-                Language.FSharp => $"#r \"{relativePath}\""
+                //Even when a user changes the current directory, loading from a relative path is not affected.
+                await kernel.SendAsync(new SubmitCode("System.IO.Directory.SetCurrentDirectory(\"..\")"));
+            }
+
+            await kernel.SendAsync(new SubmitCode("Environment.CurrentDirectory"));
+
+            var currentDirectoryName = new DirectoryInfo(Directory.GetCurrentDirectory()).Name;            
+
+            var code = language switch
+            {                
+                Language.CSharp => $"#load \"RelativeLoadingSample.csx\"",
+                Language.FSharp => $"#load \"RelativeLoadingSample.fsx\""
             };
 
             var command = new SubmitCode(code);
+            await kernel.SendAsync(command);
+
+            KernelEvents.Should()
+                        .ContainSingle<CommandSucceeded>(c => c.Command == command);
+        }
+
+        [Theory]
+        [InlineData(Language.CSharp)]
+        //Not implemented: [InlineData(Language.FSharp)]
+        public async Task it_can_load_script_files_using_load_directive_with_relative_path_after_command_changeWorkspaceDirectory(Language language)
+        {
+            var currentDirectory = Directory.GetCurrentDirectory();
+            DisposeAfterTest(() => Directory.SetCurrentDirectory(currentDirectory));
 
+            var kernel = CreateKernel(language);
+            var absolutePathOneLevelHigher = Directory.GetParent(currentDirectory).FullName;
+            await kernel.SendAsync(new ChangeWorkspaceDirectory(absolutePathOneLevelHigher));        
+
+            await kernel.SendAsync(new SubmitCode("Environment.CurrentDirectory"));
+
+            var currentDirectoryName = new DirectoryInfo(Directory.GetCurrentDirectory()).Name;
+            var relativePath = Path.GetRelativePath(absolutePathOneLevelHigher, currentDirectory);
+
+            var code = language switch
+            {
+                Language.CSharp => $"#load \"{relativePath}/RelativeLoadingSample.csx\"",
+                Language.FSharp => $"#load \"{relativePath}/RelativeLoadingSample.fsx\""
+            };
+
+            var command = new SubmitCode(code);
             await kernel.SendAsync(command);
 
             KernelEvents.Should()
```

Comment:
Same success condition as above: stdout contained `"hello!"`

Reply:
done in https://github.com/dotnet/interactive/pull/1176/commits/0dd55f30e66ea43e619e2fa75ba4cccdbbae6f6e